src/components/common/Footer/components/SocialLinks/SocialLinks.tsx

```diff
@@ -0,0 +1,12 @@
+import Link from 'next/link';
+import styles from './styles.module.scss';
+
+export const SocialLinks = () => {
+    return (
+        <div className={styles.container}>
+            <Link href='https://www.facebook.com/'><a className={`${styles.link} ${styles.fb}`} target='_blank'></a></Link>
```

Comment:
please use svgr - https://frontend-digest.com/how-to-import-svgs-into-nextjs-8ec6100e613f

Save svg icons in `common/icons/socials/...`